app/code/core/Mage/GoogleAnalytics/etc/system.xml

```diff
@@ -57,6 +57,32 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </account>
+                        <user_id translate="label">
+                            <label>User_id tracking</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>21</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment>Enable GA4 User_id tracking for logged in customers.</comment>
+                            <depends>
+                              <type>analytics4</type>
+                            </depends>
+                        </user_id>
+                        <debug translate="label">
```

Comment:
should we add a default value (to false)?

Reply:
I don't think it's necessary, config type system_config_source_yesno is false by default, also config "Enable IP anonymization" does not have an explicit value